test/extensions/quic_listeners/quiche/platform/epoll_ptr_util_impl.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include <memory>
+
+namespace epoll_server {
+
+template <typename T, typename... Args> std::unique_ptr<T> EpollMakeUniqueImpl(Args&&... args) {
```

Comment:
nit:

template <typename T, typename... Args>
using EpollMakeUniqueImpl = std::make_unique<T, Args...>;

Reply:
alias doesn't work with function template.

Reply:
Sorry about the wrong suggestion, and thanks for trying:)